packages/monitors/src/BalanceMonitor.js

```diff
@@ -147,44 +173,26 @@ class BalanceMonitor {
             this.client.getEtherBalance(monitoredAddress),
             "ETH",
             "ether",
-            18
+            num => this.toBN(num)
           )
         });
       }
     }
   }
 
-  _createLowBalanceMrkdwn(bot, threshold, tokenBalance, tokenSymbol, tokenName) {
```

Comment:
I refactored this a bit to re-use the same `_createLowBalanceMrkdwn` for all log types. you feed in a `normalizationFunction` to define the scaling done within the logger.